exporter/logicmonitorexporter/internal/traces/sender.go

```diff
@@ -44,27 +44,23 @@ func NewSender(ctx context.Context, endpoint string, client *http.Client, authPa
 func (s *Sender) SendTraces(ctx context.Context, td ptrace.Traces) error {
 	ingestResponse, err := s.traceIngestClient.SendTraces(ctx, td)
 	if err != nil {
-		return consumererror.NewPermanent(err)
-	}
-	if ingestResponse != nil {
-
-		if ingestResponse.Success {
-			return nil
-		}
```

Comment:
I don't understand why this has been removed

Reply:
Because, with the updated SDK implementation if there is non empty error, then this `success` flag in the response is always false, so no need to check that explicitly and if error is `nil` then it is always true.

Reply:
Thanks for the explanation :) LGTM!